vmdb/app/models/configured_system_foreman.rb

```diff
@@ -1,2 +1,5 @@
 class ConfiguredSystemForeman < ConfiguredSystem
+  def provider_object
+    ManageiqForeman::Host.new(configuration_manager.raw_connect, manager_ref)
```

Comment:
so, provider_object should receive the connection as a parameter when called through with_provider_object in [ProviderObjectMixin](https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/models/mixins/provider_object_mixin.rb).  Then you don't need to do the raw_connect here.